src/core/network.ts

```diff
@@ -0,0 +1,57 @@
+import { z } from "zod";
+
+import { ConsentLog, Policy, policySchema } from "@/types";
+
+type NetworkClientConfig = {
+  policiesUrl: string;
+  consentLogUrl: string;
+  userLogUrl: string;
+};
+
+/**
+ * Fetches policies and logs consents to oak-consent-api
+ */
+export class NetworkClient {
```

Comment:
Happy for a better name. Maybe `OakConsentApiClient` since it will only really work with oak-consent-api?

Reply:
Happy for you to change the name, but also happy to keep the name as it's not exposed to the consumer, and it's the only place we're where network calls are made.